src/SOS/Strike/strike.cpp

```diff
@@ -1080,7 +1081,14 @@ DECLARE_API(DumpClass)
         ParentEEClass = mtdataparent.Class;
     }
 
-    DMLOut("Parent Class:    %s\n", DMLClass(ParentEEClass));
+    if (!net9preferMT)
+    {
+        DMLOut("Parent Class:    %s\n", DMLClass(ParentEEClass));
+    }
+    else
+    {
+        DMLOut("Parent Method Table: %s\n", DMLMethodTable(mtdata.ParentMethodTable));
```

Comment:
```suggestion
        DMLOut("Parent MethodTable: %s\n", DMLMethodTable(mtdata.ParentMethodTable));
```
Nit: MethodTable is typically written without space.